shared/templates/static/ansible/accounts_root_path_dirs_no_write.yml

```diff
@@ -0,0 +1,25 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = medium
+
+- name: "Fail if user is not root"
+  fail:
+    msg: 'Root account required to read root $PATH'
+  when: ansible_user != "root"
```

Comment:
@redhatrises 
If "standard" task want to e.g. install package as non-root user, it will fail - that is not problem, it will not touch anything.
But if this task is performed as non-root user, it will change something what it should not change - it can change permission of user directories in $PATH.

I think that all tasks will be started with root permissions in real world. But this check is to make it "safer".